quadrat/inc/block-patterns.php

```diff
@@ -11,19 +11,64 @@ function quadrat_register_block_patterns() {
 
 		if ( function_exists( 'register_block_pattern_category' ) ) {
 			register_block_pattern_category(
-				'quadrat-blocks',
-				array( 'label' => __( 'Quadrat', 'quadrat-blocks' ) )
+				'quadrat',
+				array( 'label' => __( 'Quadrat', 'quadrat' ) )
 			);
 		}
 
 		if ( function_exists( 'register_block_pattern' ) ) {
 
 			register_block_pattern(
-				'quadrat-blocks/example',
+				'quadrat/the-hosts',
 				array(
-					'title'      => __( 'Alternating Grid of Latest Posts', 'quadrat-blocks' ),
-					'categories' => array( 'quadrat-blocks' ),
-					'content'    => '<!-- wp:latest-posts {"displayPostContent":true,"columns":5,"className":"is-style-quadrat-alternating-grid"} /-->',
+					'title'      => __( 'The Hosts', 'quadrat' ),
```

Comment:
The pattern names could use some help.

Reply:
> With the media + text pattern, there is a little more space at the top than the bottom. I didn't want to add a spacer because it seemed heavy-handed, but maybe it needs it.
> 
> What do you think @beafialho ?

I think it's fine leaving it as it is.

I added the patterns names on [Figma](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/?node-id=443%3A4):
<img width="533" alt="Captura de ecrã 2021-04-16, às 15 21 10" src="https://user-images.githubusercontent.com/65220155/115038419-6e3b6980-9ec7-11eb-9de4-eaa911ab1cf4.png">